posts/legacy-mainframe-migration-to-jhipster.md

```diff
@@ -0,0 +1,72 @@
+---
+authors:
+- Richard Yhip
+tags:
+- JHipster
+- Legacy Migration
+- JDL Studio
+- Microservices
+date: 2018-10-08T19:54:42+00:00
+title: "Legacy Mainframe Migration to JHipster"
+image: 
+---
+
+# Migrating a legacy COBOL mainframe application to a modern Java, Spring Boot, and Angular architecture
+
+Believe it or not, we haven't migrated all of the old legacy mainframe applications from the 1970's to the latest shiny new application platforms.  Okay, so that was a little tongue-in-cheek.  You _might_ be surprised to know how many critical features you use in your everyday life rely on mainframe functionality, however [ciation needed].
+
+Recently we here at Ippon Technologies were asked to help lead a transition from a COBOL application (that contained data from the 1800's even) to a modern microservices architecture.  We were asked to pitch in because of our famed application generator, [JHipster](https://www.jhipster.tech/).  JHipster can be useful in certain situations such as:
+* You are unfamiliar with the latest technologies and you would like to see a solid "best practices" implementation
+* You need to accelerate application development and the target application to build is some form of a [CRUDL](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) application.  You might be asking what the "L" is for at the end of "CRUDL."  It stands for listing, or pagination, which JHipster can help you with.
+* You have a large team of developers, with varying levels of expertise, and you need help standardizing development norms
+
+In our particular case, we faced all of those challenges.  We also had a problem that JHipster cannot solve that I imagine many attempts to do this will: how do you extract every business rule and other pieces of functionality from the existing legacy system?
```

Comment:
"We also had a problem that JHipster cannot solve that I imagine many attempts to do this will" -> it might be me, but I don't understand this sentence!